lib/dal/src/attribute/value/dependent_value_graph.rs

```diff
@@ -198,7 +252,7 @@ impl DependentValueGraph {
                     .value_depends_on(parent_attribute_value_id, current_attribute_value_id);
             }
         }
-
+        info!("Dependent value graph: {:?}", dependent_value_graph);
```

Comment:
Should probably drop this (could end up being a ton of data sent to the otelcol)